integration-tests/.storybook/config.js

```diff
@@ -38,4 +38,5 @@ setOptions({
   showSearchBox: false,
   downPanelInRight: true,
   sortStoriesByKind: false,
+  hierarchySeparator: /\./,
```

Comment:
@joscha We have some stories in the integration tests that are relying on the default hierarchySeparator.  I think that got broken in the 3.2.0 release, and was fixed in 3.2.1  (3.2.3 has shipped now)

The idea of using a custom is a good one - but we should probably update the story names to match.